website/blog/2022-02-22-towards-better-releases.md

```diff
@@ -0,0 +1,55 @@
+Some of these issues were quite obvious and we were aware of them for a while.
+
+There were two main improvements that were implemented since the last release:
+- Backpublish Scala versions support
+- Pre-release version for VSCode extension
+
+## Backpublish Scala versions support
+
+We haven't mentioned this in the release notes for `0.11.0`, but from that version we have changed the way Metals detect whether a specific version is supported and thanks to that we are now able to backpublish new Scala versions support.
+The support of Scala `3.1.1` Scala was added to Metals `0.11.1` using this new mechanic.
+It allows us to provide new Scala versions support much faster than before and also support Scala3-NIGTHTLY versions.
+
+The additional benefit from this approach is that it eliminates time pressure from future Metals releases.
+If you look at release notes for previous versions, almost every one brings at least one new compiler version support.
+Having this limitation and the need to provide the new release as soon as possible it always was increasing the chances to affect the quality of release. So, now with this new feature we are able to take our time for final fixes and do release with more confidence.
+
+The current state for Scala versions support is:
+- Every Metals release comes with support to all known supported Scala versions + last 5 latest Scala3-NIGHTLY versions.
+  This is applied for SNAPSHOT releases too.
+- In case new Scala version appear, the latest Metals release will receive its support automatically.
+  For example, the next Scala `3.1.2`  will be supported only by the latest Metals `0.11.1` but not by `0.11.0`.
+  That works for Scala3-NIGHTLY versions too. Metals has a scheduled daily job that publishes artifacts for newly discovered NIGHTLY versions.
+
+
+## Pre-release version for VSCode extension
+
+Another great feature that was added was the possibility to use the `pre-release` versions of Metals extension.
+If you open Metals extension page, you will find `Switch to Pre-release version` button.
+This version is like a snapshot, it's published per every change in [scalameta/metals-vscode](https://github.com/scalameta/metals-vscode) repository.
+
+It allows to test not yet released features, as well as to check if everything works fine for your workspace using the latest main branch. 
+Some issues might be observed only under the particular version of the client as was the case with the ill fated 0.11.0 release.
+
+Also, there is a new setting `Metals: Suggest Latest Upgrade` that is enabled by default for `pre-release`.
+If you have it enabled, you will receive notifications with an option to upgrade Metals-server version to the latest snapshot once a day.
+
+We hope that some brave users will start using this `pre-release` version and report issues if there will be any.
+This will help us spot problems earlier.
```

Comment:
I would add `Using pre-release versions may result in a less stable experience and it is not intended for beginners.` somewhere. Pre-release is great but I think we didn't warn users enough about potential issues.

Reply:
Added into `Notice`